brave/brave-core
# Refactor/cleanup search promotion ui

Resolves

fix https://github.com/brave/brave-browser/issues/36499
fix https://github.com/brave/brave-browser/issues/37531

Removed deprecated banner type A and button type promotions.
And fixed omnibox search promotion layout regression.



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Launch brave with `--enable-features=BraveSearchOmniboxBanner:banner_type/type_A` or `--enable-features=BraveSearchOmniboxButton`
2. Change default search provider to non-brave search
3. Type anything and check promotion entry is not shown in omnibox popup
4. Relaunch brave with `--enable-features=BraveSearchOmniboxBanner:banner_type/type_B`
5. Check promotion entry is shown in omnibox
